packages/test-matchers/toEqualArrayBuffer.ts

```diff
@@ -1,3 +1,5 @@
+import { bytesEqual } from '@solana/codecs-core';
```

Comment:
Dang it. On a fresh install (see the CI run) this fails.  
  
\> Cannot find module '@solana/codecs-core' from '../test-matchers/toEqualArrayBuffer.ts'  
  
I can't remember if our private packages have a restriction that makes it impossible to take on dependencies. Do you have time to troubleshoot this (or just revert it to the way it was)